frontend/fakenoteprovider.js

```diff
@@ -19,7 +19,7 @@ class FakeNoteProvider extends NoteProvider {
     constructor() {
         super();
         this.#notes = new Map([
-            ["Sample", new Note("Sample", "# Sample Note\n")]
+            ["Sample", new Note("Sample", "# Sample Note\n", ["Sample"])]
```

Comment:
I think the content should differ whether it's a name, content or tag. Also at this point it would be great to demonstrate, that multiple tags are possible and how it should be formatted.

Reply:
You can do all this by using the UI. This class is named `FakeNoteProvider` by purpose.